x-pack/auditbeat/module/system/_meta/config.yml.tmpl

```diff
@@ -1,14 +1,17 @@
 {{ if .Reference -}}
 {{ end -}}
 - module: system
-  
+
   metricsets:
     - host
     - packages
     - processes
+    - user
```

Comment:
The inclusion of `user` in the config file should be conditional on the `.GOOS` value. We don't want it showing up in packages on non-Linux.

Reply:
Done.